### DISCUSSION TITLE
ERROR: No loader is configured for ".node" files: node_modules/node-pty/build/Release/pty.node

### DISCUSSION THREAD

**Discussion by 2687545691** [Microsoft/node-pty#738] [Q&A]
## Environment details

- OS:Windows
- OS version:11
- node-pty version:lastest

## Issue description
Mine is an electron+vite+ts+react project, and after integrating the node-pty module and configuring it according to the readme document, the project gets an error “ERROR: No loader is configured for ".node" files: node_modules/node-pty/build/Release/pty.node”.

How to solve this mistake?

**Comment by qarlosalberto** [comment]
Same here.

**Comment by Tyriar** [comment]
This looks related to your bundler
